### PR TITLE
Update dependencies due to security vulnerabilities

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

Update `elliptic` and `dot-prop` dependencies due to security vulnerabilities
See https://www.npmjs.com/advisories/1547
See https://www.npmjs.com/advisories/1213

As a temporary solution until we can republish our packages, I've set custom resolutions in yarn.

TODO to completely eliminate the vulnerable versions of `elliptic <= 6.5.2`:
- [ ] republish `contractkit` and `utils` which depends on fixed version `elliptic@6.5.2`
- [ ] update `ethers` which also which depends on fixed version `elliptic@6.3.3`

### Other changes

None

### Tested

Waiting for test to pass on CircleCI.

### Related issues

Discussed on Slack

### Backwards compatibility

Should be 👀

### CODE REVIEW

File: packages/protocol/package.json [hunk 1]
```diff
@@ -43,7 +43,6 @@
     "@0x/sol-profiler": "^3.0.0",
     "@0x/sol-trace": "^2.0.16",
     "@0x/subproviders": "^5.0.0",
-    "@openzeppelin/upgrades": "^2.8.0",
```
Comment (gastonponti) [inline packages/protocol/package.json:46]: why this one? (it doesn't seem to be part of the same issue)

Comment (jeanregisser) [reply packages/protocol/package.json:46]: This line was duplicated (see below) and when I used `yarn upgrade-interactive` it rewrote it (and others).

Comment (gastonponti) [reply packages/protocol/package.json:46]: Ah, you are right!

**Review by gastonponti — APPROVED** [review]

**Review by codyborn — APPROVED** [review]

**Review by jmrossy — APPROVED** [review]

**Comment by jmrossy** [comment]
RE ethers, we don't have any direct dependencies on it so we may be able to solve that transitive dep issue with a yarn resolution config

**Pull request merged**